File: app/src/main/java/com/song/deviceinfo/ui/thermal/ThermalViewModel.java
package com.song.deviceinfo.ui.thermal;

import com.song.deviceinfo.info.ThermalInfo;
import com.song.deviceinfo.ui.base.NormalViewModel;

import java.util.List;

import androidx.core.util.Pair;

/**
 * Created by chensongsong on 2020/5/27.
 */
public class ThermalViewModel extends NormalViewModel {

    public List<Pair<String, String>> getThermalInfo() {
        return ThermalInfo.getThermalInfo();
    }
}

File: app/src/main/java/com/song/deviceinfo/ui/thermal/ThermalViewModel.kt
package com.song.deviceinfo.ui.thermal

import androidx.core.util.Pair
import com.song.deviceinfo.info.ThermalInfo
import com.song.deviceinfo.ui.base.NormalViewModel

/**
 * Created by chensongsong on 2020/5/27.
 */
class ThermalViewModel : NormalViewModel() {
    val thermalInfo: List<Pair<String, String>>
        get() = ThermalInfo.thermalInfo
}
